mqtt/mqtt-broker/src/broker.rs

```diff
@@ -32,7 +30,9 @@ macro_rules! try_send {
 pub struct Broker<N, Z>
 where
     N: Authenticator,
+    N::Error: Display + Into<Error>,
     Z: Authorizer,
+    Z::Error: Display + Into<Error>,
```

Comment:
We only use this error for logging? Do we ever bubble this error out or take action on it?